code/modules/hydroponics/grown.dm

```diff
@@ -166,7 +166,6 @@
 	var/obj/item/T
 	if(trash)
 		T = generate_trash()
-	qdel(src)
-	if(trash)
 		user.put_in_hands(T)

```

Comment:
@ChangelingRain which proc would I use for that, there's a couple of unequips now and I don't want to choose the wrong one.